.nvmrc

```diff
@@ -0,0 +1 @@
+18.13.0
```

Comment:
```suggestion
v18.13.0

```

Reply:
In correct format! Please check the output for the command `node --version`, as this should be the same as that. That is, it should be `v18.13.0` instead of `18.13.0`.